sda/cmd/s3inbox/proxy.go

```diff
@@ -168,6 +168,35 @@ func (p *Proxy) allowedResponse(w http.ResponseWriter, r *http.Request, token jw
 		}
 	}
 
+	// check if the file already exists, in that case send an overwrite message,
+	// so that the FEGA portal is informed that a new version of the file exists.
+	exist := p.checkFileExists(r.URL.Path)
+	if exist {
+		log.Debug("create rewrite message")
+		event := Event{}
+		var err error
+		event.Operation = "remove"
+		event.Filepath = rawFilepath
+		event.Username = username
```

Comment:
If you import `github.com/neicnordic/sensitive-data-archive/internal/schema` you will get access to the JSON schemas and the structs so you can validate the outgoing payload.